monasca-alarms/Dockerfile

```diff
@@ -12,12 +14,8 @@ ENV MONASCA_WAIT_FOR_API=true \
   OS_USERNAME=mini-mon \
   OS_USER_DOMAIN_NAME=Default
 
-RUN apk add --no-cache \
-    python py2-pip py2-netaddr py2-yaml py2-jinja2 && \
-  apk add --no-cache --virtual build-dep \
-    git python-dev make g++ linux-headers && \
-  pip install python-monascaclient && \
-  rm -rf /root/.cache/pip && \
+RUN apk add --no-cache py2-netaddr py2-yaml py2-jinja2 && \
+  apk add --no-cache --virtual build-dep && \
   apk del build-dep
```

Comment:
Hmm, this seems a bit odd. If `build-dep` is empty we can probably skip adding/deleting it entirely.